models/mongoDB/index.js

```diff
@@ -6,40 +6,83 @@ mongoose.connect('mongodb://localhost/qa', (err) => {
   }
 });
 
+const questionsImportSchema = new mongoose.Schema({
+  question_id: { type: Number, unique: true },
+  product_id: { type: Number, index: true },
+  question_body: { type: String, minLength: 1, maxLength: 1000 },
+  question_date: { type: String },
+  asker_name: { type: String, minLength: 1, maxLength: 60 },
+  asker_email: { type: String, minLength: 1, maxLength: 60 },
+  reported: { type: Number, max: 1 },
+  question_helpfulness: { type: Number },
+});
+
+const answersImportSchema = new mongoose.Schema({
+  id: { type: Number, unique: true },
+  question_id: { type: Number, index: true },
+  body: { type: String, minLength: 1, maxLength: 1000 },
+  date: { type: String },
+  answerer_name: { type: String, minLength: 1, maxLength: 60 },
+  answerer_email: { type: String, minLength: 1, maxLength: 60 },
+  reported: { type: Number, max: 1 },
+  helpful: { type: Number },
+});
+
+const answersPhotosImportSchema = new mongoose.Schema({
+  id: { type: Number, unique: true },
+  answer_id: { type: Number, index: true },
+  url: { type: String },
+});
+
+// for ETL import
+const QuestionImport = mongoose.model('QuestionImport', questionsImportSchema);
+const AnswerImport = mongoose.model('AnswerImport', answersImportSchema);
+const AnswerPhotoImport = mongoose.model('AnswerPhotoImport', answersPhotosImportSchema);
+
+// for CRUD operations after combining imports
 const questionsSchema = new mongoose.Schema({
   question_id: { type: Number, unique: true },
-  product_id: { type: Number },
-  question_body: { type: String },
+  product_id: { type: Number, index: true },
+  question_body: { type: String, minLength: 1, maxLength: 1000 },
   question_date: { type: String },
-  asker_name: { type: String },
-  asker_email: { type: String },
-  reported: { type: Number },
+  asker_name: { type: String, minLength: 1, maxLength: 60 },
+  asker_email: { type: String, minLength: 1, maxLength: 60 },
+  reported: { type: Number, max: 1 },
   question_helpfulness: { type: Number },
+  answers: { type: Object, default: {} },
+}, { minimize: false });
+
+const lastQuestionIdSchema = new mongoose.Schema({
+  question_id: { type: Number, unique: true },
 });
 
 const answersSchema = new mongoose.Schema({
-  answer_id: { type: Number, unique: true },
-  question_id: { type: Number },
-  answer_body: { type: String },
-  answer_date: { type: String },
-  answerer_name: { type: String },
-  answerer_email: { type: String },
-  reported: { type: Number },
-  answer_helpfulness: { type: Number },
+  id: { type: Number, unique: true },
+  question_id: { type: Number, index: true },
+  body: { type: String, minLength: 1, maxLength: 1000 },
+  date: { type: String },
+  answerer_name: { type: String, minLength: 1, maxLength: 60 },
+  answerer_email: { type: String, minLength: 1, maxLength: 60 },
+  reported: { type: Number, max: 1 },
+  helpful: { type: Number },
+  photos: [String],
 });
 
```

Comment:
Love the refactoring and further specification (indexes, uniqueness, length, etc) for optimization for your models.